account_bank_statement_import_paypal/wizards/account_bank_statement_import_paypal.py

```diff
@@ -156,7 +156,7 @@ def _post_process_statement_line(self, raw_lines):
             # transaction of currency change if yes merge the transaction
             # as for odoo it's only one line
             cline = currency_change_lines.get(line['origin_transaction_id'])
-            if cline:
+            if cline and (cline['amount'] * line['amount'] > 0):
```

Comment:
I'm slightly puzzled by this line. It looks like you encountered a statement line with the currency amount having the wrong sign. Can you clarify this a bit, or even better, add a test for this situation?